index.html

```diff
@@ -52,18 +52,18 @@ <h2 id="recent">My Recent Works</h2>
                 <div class="box"></div>
             </div>
 
-            <div class="third-container">
+            <div class="third-container" id="third-container">
                 <img src="images/placeholder.png" alt="placeholder" class="img2">
                 <div class="third-container-2">
                     <h3 id="multi">Multi-Post Sories</h3>
                     <p id="par2">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
                     <ul class="langs">
-                        <li><button class="langbuttons">css</button></li>
-                        <li><button class="langbuttons">html</button></li>
-                        <li><button class="langbuttons">bootstrap</button></li>
-                        <li><button class="langbuttons">Ruby</button></li>
+                        <li><button type="button" class="langbuttons">css</button></li>
+                        <li><button type="button" class="langbuttons">html</button></li>
+                        <li><button type="button" class="langbuttons">bootstrap</button></li>
+                        <li><button type="button" class="langbuttons">Ruby</button></li>
                     </ul>
-                    <button class="project">See Project</button>
+                    <button type="button" class="project" onclick="popupwindow()">See Project</button>
```

Comment:
- [ ] Kindly populate the project section dynamically with a javascript code that will be looping through the project array.
> eg:

```
projects.forEach((project) => {
     section.append(...)
})
```

Reply:
hi @aimemalaika, first of all thank you for your review. I didn't understood that all the card/projects html code needed to be in the js file. I managed to do so please check it again.